webpack.config.js

```diff
@@ -16,14 +17,36 @@ const NODE_MODULES = path.resolve(__dirname, 'node_modules');
 const OUT_DIR = path.resolve(__dirname, '_site');
 const ROOT_DIR = path.resolve(__dirname, 'public');
 
+function parseExtensions() {
+  const readmePath = path.resolve(
+    __dirname,
+    'node_modules/simple-icons/README.md',
+  );
+  const body = fs.readFileSync(readmePath, 'utf8');
+  const extRegExp = /## Third-Party Extensions\n\n(.*)\[slug\]/gs;
+  return Array.from(body.match(extRegExp))[0]
+    .split('\n')
+    .filter((line) => line.startsWith('| ['))
+    .map((line) => {
+      const module = line.split(' | ')[0];
+      const author = line.split(' | ')[1];
+      return {
+        nameModule: module.match(/\[(.*?)\]/gs)[0].slice(1, -1),
+        urlModule: module.match(/\((.*?)\)/gs)[0].slice(1, -1),
+        nameAuthor: author.match(/\[(.*?)\]/gs)[0].slice(1, -1),
+        urlAuthor: author.match(/\((.*?)\)/gs)[0].slice(1, -1),
+      };
+    });
```

Comment:
I'm not really sure if this is stable because `slug` content can change in the future. Also, can be simplified:

```suggestion
  return body.split("## Third-Party Extensions\n\n")[1].split("\n\n")[0]
    .split('\n')
    .slice(2)
    .map((line) => {
      const [module, author] = line.split(' | ');
      return {
        nameModule: /\[(.*?)\]/.exec(module)[1],
        urlModule: /\((.*?)\)/.exec(module)[1],
        nameAuthor: /\[(.*?)\]/.exec(author)[1],
        urlAuthor: /\((.*?)\)/.exec(author)[1],
      };
    });
```